airbyte-integrations/connectors/source-mongodb-v2/src/main/java/io/airbyte/integrations/source/mongodb/cdc/MongoDbCdcInitializer.java

```diff
@@ -119,7 +119,7 @@ public List<AutoCloseableIterator<AirbyteMessage>> createCdcIterators(
       AirbyteTraceMessageUtility.emitAnalyticsTrace(cdcCursorInvalidMessage());
       if (config.shouldFailSyncOnInvalidCursor()) {
         throw new ConfigErrorException(
-            "Saved offset is not valid. Please reset the connection, and then increase oplog retention or reduce sync frequency to prevent his from happening in the future. See https://docs.airbyte.com/integrations/sources/mongodb-v2#mongodb-oplog-and-change-streams for more details");
+            "Saved offset is not valid. Please reset the connection, and then increase oplog retention and/or increase sync frequency to prevent his from happening in the future. See https://docs.airbyte.com/integrations/sources/mongodb-v2#mongodb-oplog-and-change-streams for more details");
```

Comment:
Question: I have a bunch of tables which are rarely used but we use Airbyte to sync them.
By definition, these tables will have an outdated oplog resume token that would not exist.
Shouldnt the expected behaviour be trigerring a full rebuild in that case?

Reply:
An oplog resume token being rotated out can lead to data loss since deletes aren't properly captured. In addition, it can lead to higher cloud costs due to consumption of credits. 

If you do want to rebuild you can : 
1. Move the tables that are rarely used to a new connection. 
2. In that connection, you change the invalid cdc cursor behavior to "Re-sync Data". This will change it to the previous behavior of rebuilding

Reply:
The definition was in the `source` and not in the connection 😜